source/manual/datalabs-content-store.html.md

```diff
@@ -0,0 +1,52 @@
+---
+owner_slack: "#govuk-data-labs"
+title: Use Content Store
+section: Data Labs Team Manual
+layout: manual_layout
+parent: "/manual.html"
+---
+
+The [Content Store](https://docs.publishing.service.gov.uk/apps/content-store.html) is a MongoDB database of almost all the content published on GOV.UK.
+
+The Content Store does not have all GOV.UK content. The Content Store has the content itself, but does not include dynamic elements such as top links on taxon pages, navigation elements, or search result pages.
+
+Contact a GOV.UK Data Labs software developer through the [GOV.UK Data Labs slack channel](https://gds.slack.com/archives/CHR4UQKU4) for more information.
+
+Consider using the [GOV.UK mirror](link) if you need a more representative data source of what users actually see on the website.
+
+## Get access to the Content Store
+
+To access the Content Store, you download a copy of the production version of the Content Store and query that local version using Docker.
+
+1. Sign into AWS. See the [GOV.UK Developer Docs on getting AWS access](https://docs.publishing.service.gov.uk/manual/get-started.html#7-get-aws-access) for more information.
+
+1. Go to the [`govuk-integration-database-backups` AWS S3 bucket](https://s3.console.aws.amazon.com/s3/buckets/govuk-integration-database-backups?prefix=mongo-api%2F&region=eu-west-1) and then go to the `mongo-api` folder.
```

Comment:
I got stuck at this step. Does one have to assume the right role to do this first? I think we had some guidance somewhere of how to do this.

I've found some but it doesn't quite do the trick, I'll ask on Slack to see if anyone remembers who wrote this guidance.

Reply:
I've started a slack thread here https://gds.slack.com/archives/CHR4UQKU4/p1627486067033800

Reply:
I put @maxf advice in this [doc](https://docs.google.com/document/d/12RaZP2fZulleF8tj6eJtnBL_1mXlEMZ1ZbUaCBH7Zbg/edit#heading=h.r152rsp6s0bz)

Reply:
thanks, have amended